src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/InputNeedBracesPatternMatchingForSwitch.java

```diff
@@ -14,18 +14,19 @@ public class InputNeedBracesPatternMatchingForSwitch {
 
     void testSwitchStatements(Object o) {
         switch (o) {
-            case Integer i when (i == 0) :  // violation
+            case Integer i when (i == 0) :  // violation, 'case' construct must use '{}'s
```

Comment:
violation message should have wrapping `' '` , please update for all.
like you did at `// violation below ''case' construct must use '{}'s'`

Reply:
> violation message should have wrapping `' '` , please update for all. like you did at `// violation below ''case' construct must use '{}'s'`

Done